packages/applicant/src/pages/organisation/index.page.tsx

```diff
@@ -191,4 +124,122 @@ const ManageOrganisationDetails: FC<OrganisationIndexPageProps> = ({
   );
 };
 
+type OrganisationDataRowProps = {
+  organisationDetail: OrganisationDetails;
+  noBorderBottom?: boolean;
+};
+
+const OrganisationDataRow = ({
+  organisationDetail,
+  noBorderBottom,
+}: OrganisationDataRowProps) => (
+  <div
+    className={`govuk-summary-list__row ${
+      noBorderBottom ? 'border-bottom-0-important' : ''
+    }`}
+  >
+    <dt
+      className="govuk-summary-list__key"
+      data-cy={`cy-organisation-details-${organisationDetail.label}`}
+    >
+      {organisationDetail.label}
+    </dt>
+    {organisationDetail.id === 'organisationAddress' ? (
+      <ProcessAddress
+        data={organisationDetail.value}
+        id={organisationDetail.id}
+        cyTag={organisationDetail.label}
+      />
+    ) : (
+      <dd
+        className="govuk-summary-list__value"
+        id={organisationDetail.id}
+        data-cy={`cy-organisation-value-${organisationDetail.label}`}
+      >
+        {organisationDetail.value ? organisationDetail.value : '-'}
+      </dd>
+    )}
+    <dd className="govuk-summary-list__actions">
+      <Link href={organisationDetail.url}>
+        <a
+          className="govuk-link govuk-link--no-visited-state"
+          data-cy={`cy-organisation-details-navigation-${organisationDetail.id}`}
+        >
+          {organisationDetail.status}
+          <span className="govuk-visually-hidden">
+            {organisationDetail.url}
+          </span>
+        </a>
+      </Link>
+    </dd>
+  </div>
+);
+
+type OrganisationData = {
+  typeOfOrganisationRow: OrganisationDetails;
+  generalOrganisationRows: OrganisationDetails[];
+};
+
+const getOrganisationData = (
+  organisationData: GrantApplicantOrganisationProfile,
+  { isIndividual, isNonLimitedCompany }: Record<string, boolean>
+): OrganisationData => {
+  const typeOfOrganisationRow = {
+    id: 'organisationType',
+    label: `Type of ${isIndividual ? 'application' : 'organisation'}`,
+    value: isIndividual ? 'Applying as an individual' : organisationData?.type,
```

Comment:
```suggestion
    value: isIndividual ? 'I am applying as an individual' : organisationData?.type,
```
but again this should be a constant